docker-compose.yaml

```diff
@@ -1,129 +1,117 @@
+    image: postgres:14-alpine
+    restart: always
+    expose:
+      - '5432'
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
 
-  # overworld-db:
-  #   container_name: overworld-db
-  #   image: postgres:14-alpine
-  #   restart: always
-  #   expose:
-  #     - "5432"
-  #   environment:
-  #     - POSTGRES_USER=postgres
-  #     - POSTGRES_PASSWORD=postgres
+  overworld-db:
+    container_name: overworld-db
+    image: postgres:14-alpine
+    restart: always
+    expose:
+      - '5432'
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
 
-  # bugfinder-db:
-  #   container_name: bugfinder-db
-  #   image: postgres:14-alpine
-  #   restart: always
-  #   expose:
-  #     - "5432"
-  #   environment:
-  #     - POSTGRES_USER=postgres
-  #     - POSTGRES_PASSWORD=postgres
+  bugfinder-db:
+    container_name: bugfinder-db
+    image: postgres:14-alpine
+    restart: always
+    ports:
+      - '5432:5432'
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
 
   #backends
-  # login-backend:
-  #   container_name: login-backend
-  #   image: ghcr.io/gamify-it/login-backend:main
-  #   restart: always
-  #   pull_policy: always
-  #   expose:
-  #     - "80"
-  #   environment:
-  #     - JWT_KEY=SecretKEY
-  #     - JWT_VALIDITY_DURATION=720h
-  #     - POSTGRES_URL=postgresql://postgres:postgres@login-db:5432/
-
-  # overworld-backend:
-  #   container_name:   overworld-backend
-  #   image: ghcr.io/gamify-it/overworld-backend:main
-  #   restart: always
-  #   pull_policy: always
-  #   expose:
-  #     - "80"
-  #   depends_on:
-  #     - overworld-db
-  #   environment:
-  #     - POSTGRES_USER=postgres
-  #     - POSTGRES_PASSWORD=postgres
-  #     - POSTGRES_URL=postgresql://overworld-db:5432/postgres
+  login-backend:
+    container_name: login-backend
+    image: ghcr.io/gamify-it/login-backend:main
+    restart: always
+    pull_policy: always
+    expose:
+      - '80'
+    environment:
+      - JWT_KEY=SecretKEY
+      - JWT_VALIDITY_DURATION=720h
+      - POSTGRES_URL=postgresql://postgres:postgres@login-db:5432/
 
-  # bugfinder-backend:
-  #   container_name: bugfinder-backend
-  #   image: ghcr.io/gamify-it/bugfinder-backend:main
-  #   restart: always
-  #   pull_policy: always
-  #   expose:
-  #     - "80"
-  #   depends_on:
-  #     - bugfinder-db
-  #   environment:
-  #     - POSTGRES_USER=postgres
-  #     - POSTGRES_PASSWORD=postgres
-  #     - POSTGRES_URL=postgresql://bugfinder-db:5432/postgres
+  overworld-backend:
+    container_name: overworld-backend
+    image: ghcr.io/gamify-it/overworld-backend:main
+    restart: always
+    pull_policy: always
+    expose:
+      - '80'
+    depends_on:
+      - overworld-db
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_URL=postgresql://overworld-db:5432/postgres
 
+  ## TODO: insert bugfinder backend when ready
```

Comment:
cant test until bugfinder backend is inserted... (pls do so in all relevante compose files